packages/trader/src/Utils/Validator/validator.ts

```diff
@@ -0,0 +1,165 @@
+import { template } from '_common/utility';
+import { getPreBuildDVRs } from '@deriv/shared';
+import Error from './errors';
+import { getValidationRules } from 'Stores/Modules/Trading/Constants/validation-rules';
+import { TTradeStore } from 'Types';
```

Comment:
This type will be available once this PR is merged to parent branch: https://github.com/kate-deriv/deriv-app/pull/4